examples/rig-kafka-pub-sub-example/example/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
@kevinbader @krekac does this file feel like it should be in the gitignore? I don't know mvn well enough...

Reply:
I updated the RIG .gitignore for these things, hoping it works properly.

Reply:
Gradle is considered spiritual successor to Maven. And with Gradle it is recommended for projects to use wrapper to ensure consistent version across developers and machines. It is part of source control (the wrapper files). Therefore, for Maven, I personally think it would hold true.

Though since you put the Maven package step inside the Dockerfile, executed via a base image that already contains the appropriate JDK and Maven version, I'd not be too worried about above.